md2notion/upload.py

```diff
@@ -115,34 +118,49 @@ def filesFromPathsUrls(paths):
             fileLike.name = path
             yield (path, fileName, fileLike)
         else:
-            for path in glob.glob(path, recursive=True):
+            globPaths = glob.glob(path, recursive=True)
+            if not globPaths:
+                raise RuntimeError(f'No file found for glob {path}')
+            for path in globPaths:
                 with open(path, "r", encoding="utf-8") as file:
                     yield (path, os.path.basename(path), file)
 
-if __name__ == "__main__":
-    import argparse
-    import sys
-    from notion.block import PageBlock
-    from notion.client import NotionClient
-
+def cli(argv):
     parser = argparse.ArgumentParser(description='Uploads Markdown files to Notion.so')
     parser.add_argument('token_v2', type=str,
                         help='the token for your Notion.so session')
     parser.add_argument('page_url', type=str,
                         help='the url of the Notion.so page you want to upload your Markdown files to')
     parser.add_argument('md_path_url', type=str, nargs='+',
                         help='A path, glob, or url to the Markdown file you want to upload')
+    parser.add_argument('--create', action='store_const', dest='mode', const='create',
+                        help='Create a new child page (default)')
+    parser.add_argument('--append', action='store_const', dest='mode', const='append',
```

Comment:
If I understand this right, the `--append` option still works, but the behavior is different from #10. Instead of appending markdown nodes to an existing page, it appends a new child page with potentially the same title. I'm not sure this is what @aduckworth1969 wanted in #9.

Reply:
Hmm, it should append to the existing page (the page URL passed on the CLI)? It passes `uploadPage` directly instead of creating a new one for each md file/url.

This is what the test case tests as well unless I really messed something up.